storage.tf

```diff
@@ -13,6 +13,7 @@ resource "google_storage_bucket" "deployment_utils" {
   storage_class               = "STANDARD"
   uniform_bucket_level_access = true
   provider                    = google.offensive-pipeline
+  force_destroy = true
```

Comment:
Becasue I changed the migration process where the backup is copied directly to the deployment's created bucket and read from there. Since the backup archive doesnt have a resource reference, the bucket would not destry